.github/workflows/trunk-hourly-e2e-tests.yml

```diff
@@ -126,9 +125,13 @@ jobs:
 
       - run: ls -l ./cypress/reports/mochareports/
       - name: Upload results
-        run: ./.trunk-ci-analytics/bin --token=${{ secrets.TRUNK_DEBUGGER_TOKEN }}  --input="cypress/reports/**/*junit*.xml"
+        uses: trunk-io/trunk-analytics-uploader@v0.1.0
+        with:
+          junit_paths: cypress/reports/**/*junit*.xml
+          org_url_slug: trunk-staging-org
+          token: ${{ secrets.TRUNK_API_TOKEN }}
```

Comment:
Why were we using the `TRUNK_DEBUGGER_TOKEN ` earlier? Is the `TRUNK_API_TOKEN` not set up?

Reply:
This is what old binary expected to use. We can't assume how token env var will be named in our users setup, that way we can always pass `TRUNK_DEBUGGER_TOKEN` or any other value with `--token` key.